tests/python_client/milvus_client/test_milvus_client_collection.py

```diff
@@ -386,6 +429,92 @@ def test_milvus_client_collection_self_creation_multiple_vectors(self):
         if self.has_collection(client, collection_name)[0]:
             self.drop_collection(client, collection_name)
 
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_milvus_client_create_collection_dup_name(self):
+        """
+        target: test create collection with same name
+        method: create collection with same name with same default params
+        expected: collection properties consistent
+        """
+        client = self._client()
+        collection_name = cf.gen_collection_name_by_testcase_name()
+        # 1. create collection
+        self.create_collection(client, collection_name, default_dim)
+        # 2. create collection with same params
+        self.create_collection(client, collection_name, default_dim)
+        self.drop_collection(client, collection_name)
+
+    @pytest.mark.tags(CaseLabel.L2)
+    def test_milvus_client_collection_dup_name_with_desc(self):
+        """
+        target: test creating duplicate named collections with description consistency
+        method: 1. create collection with custom description
+                2. create same collection again with identical schema and description
+                3. verify description remains consistent between both operations
+        expected: description property should be identical across multiple collection references
+        """
+        client = self._client()
+        collection_name = cf.gen_collection_name_by_testcase_name()
+        description = "test collection description"
+        # 1. Create schema with description
+        schema = self.create_schema(client, enable_dynamic_field=False, description=description)[0]
+        schema.add_field("id", DataType.INT64, is_primary=True, auto_id=False)
+        schema.add_field("embeddings", DataType.FLOAT_VECTOR, dim=default_dim)
+        # 2. Create collection first time with description
+        self.create_collection(client, collection_name, schema=schema)
+        # Get description from first "reference"
+        collection_info_1 = self.describe_collection(client, collection_name)[0]
+        description_1 = collection_info_1.get("description", "")
+        # 3. Create collection again with same name (should succeed)
+        self.create_collection(client, collection_name, schema=schema)
+        # Get description from second "reference"  
+        collection_info_2 = self.describe_collection(client, collection_name)[0]
+        description_2 = collection_info_2.get("description", "")
+        # 4. Verify descriptions are consistent
+        assert description_1 == description_2 == description
+        self.drop_collection(client, collection_name)
+
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_milvus_client_create_collection_dup_name_same_schema(self):
```

Comment:
I think we can remove this case now, because the milvus client sdk does not return any object to client.